server/controller/comments/getAllCommentForAPost.go

```diff
@@ -0,0 +1,45 @@
+package comments
+
+import (
+	"campusburn-backend/dbConnection"
+	"campusburn-backend/model"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+/*
+FLOW TO GET ALL THE COMMENTS FOR A POST:
+---------------------------------------
+1. Get the post Id from the request.
+2. Find all the comments for this post from the database.
+3. If any error while finding the comments, then return error response.
+4. Return the success response with all comments as the data.
+
+*/
+
+func GetAllCommentForAPost(c *fiber.Ctx) error {
+
+	var post []model.Post
+	var postId = c.Params("postId")
```

Comment:
We won't be getting the `postId` from the params, we will be sending it as `json` data from the frontend. So, you need to create a struct here to accept the `json`.

Reply:
Okay, I will modify it. Also, what other data are there in the `json` along with `postId`?

Reply:
> Okay, I will modify it. Also, what other data are there in the `json` along with `postId`?

`postId` is fine for now.

Reply:
and also what should the route name be you want? just `AllComments` or there should id of the post after `AllComments`?
although, I am using `json` to get the `postId`

Reply:
> and also what should the route name be you want? just `AllComments` or there should id of the post after `AllComments`? although, I am using `json` to get the `postId`

for now let's keep the route `/allComments`